tests/account/test_api.py

```diff
@@ -525,6 +526,31 @@ async def test_login(
     assert await resp.json() == snapshot
 
 
+async def test_login_system_user(
+    mongo: Mongo,
+    spawn_client: ClientSpawner,
+    snapshot,
+):
+    client = await spawn_client()
+
+    await mongo.users.insert_one(
+        {
+            "user_id": "abc123",
+            "handle": "foobar",
+            "password": hash_password("p@ssword123"),
+            "type": "bot",
+        },
+    )
```

Comment:
Doesn't the `fake` fixture work for this?

If not, please update to support `type` if that is what you need for this test.